MapleServer2/Servers/Game/FieldManager.cs

```diff
@@ -309,6 +321,16 @@ public bool RemoveCube(IFieldObject<Cube> cube)
             return State.RemoveCube(cube.ObjectId);
         }
 
+        public void AddInstrument(IFieldObject<Instrument> instrument)
+        {
+            State.AddInstrument(instrument);
+        }
+
+        public bool RemoveInstrument(IFieldObject<Instrument> instrument)
```

Comment:
And return methods with a single line also can be express with lambda 
```suggestion
        public bool RemoveInstrument(IFieldObject<Instrument> instrument) => State.RemoveInstrument(instrument.ObjectId);
```

Reply:
i mean if you want to do return methods that only have a line to be expressed with lambda you will have alot to edit since there are quite a few.